turbopack/crates/turbopack-browser/src/ecmascript/content.rs

```diff
@@ -106,24 +105,22 @@ impl EcmascriptBrowserChunkContent {
         // When the runtime executes (see the `evaluate` module), it will pick up and
         // register all pending chunks, and replace the list of pending chunks
         // with itself so later chunks can register directly with it.
-        writedoc!(
+        write!(
             code,
-            r#"
-                (globalThis.TURBOPACK = globalThis.TURBOPACK || []).push([{script_or_path}, {{
-            "#
+            "(globalThis.TURBOPACK ||= []).push([{script_or_path},"
```

Comment:
```suggestion
            "(globalThis.TURBOPACK || (globalThis.TURBOPACK = [])).push([{script_or_path},"
```

for < es2020 compat?